src/core/index.ts

```diff
@@ -11,6 +11,21 @@ import { FormSubmission } from "./drive/form_submission"
 const session = new Session
 const { navigator } = session
 export { navigator, session, PageRenderer, PageSnapshot }
+export {
+  TurboBeforeCacheEvent,
+  TurboBeforeRenderEvent,
+  TurboBeforeVisitEvent,
+  TurboClickEvent,
```

Comment:
I'm not sure what would be better: `BeforeCacheEvent`, `BeforeRenderEvent`, etc internally and exporting them here as `Turbo`-prefixed versions, or sticking with the current style of qualifying them with `Turbo`.

Reply:
Or even export them without the `Turbo` prefix? `FrameElement` and `StreamElement` don't have it.